src/main/java/manageezpz/logic/parser/CliSyntax.java

```diff
@@ -9,7 +9,11 @@
     public static final Prefix PREFIX_NAME = new Prefix("n/");
     public static final Prefix PREFIX_PHONE = new Prefix("p/");
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
+    public static final Prefix PREFIX_DESCRIPTION = new Prefix("desc/");
+    public static final Prefix PREFIX_DATETIME = new Prefix("by/");
+    public static final Prefix PREFIX_TIME = new Prefix("at/");
     public static final Prefix PREFIX_TODO = new Prefix("/todo");
     public static final Prefix PREFIX_EVENT = new Prefix("/event");
```

Comment:
just a nitpick maybe but why the sudden format change in the prefix? prev was "by\" for example but now it's "/todo".

Reply:
Hi aaron, there is some issues with the PR being reflected but not to worry. We would still stick with the format of "type/", "by/", "at/" etc
eg: addTask todo/ desc/read book
Sorry for any confusion!